dataset/dataset_test.go

```diff
@@ -847,6 +852,22 @@ func TestClient_GetOptions(t *testing.T) {
 			})
 		})
 
+		Convey("when GetOptions is called with a list of IDs containing an option with special characters", func() {
+			q := QueryParams{offset, limit, []string{"90+"}}
```

Comment:
For robustness, should we not be setting the keys to their values?

`q := QueryParams{Offset: offset, Limit: limit, Options: []string{"90+"}}`

Otherwise you are relying on the order which is ambiguous and can easily break when this is extended to cover more query parameters.

Reply:
Good point, I will also fix the rest of tests that do the same.